fixtures/u2_class_empty.rb.uast

```diff
@@ -8,21 +8,27 @@ module {
 .  .  0: class {
 .  .  .  Roles: Declaration,Identifier,Statement,Type
 .  .  .  TOKEN "Testcls1"
```

Comment:
This should be an `Identifier` in `.sem.uast`.

Reply:
Fixed in next commit.